arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -0,0 +1,776 @@
+      "type": "object",
+      "description": "The consistency policy for the DocumentDB database account.",
+      "properties": {
+        "defaultConsistencyLevel": {
+          "description": "The default consistency level and configuration settings of the DocumentDB account.",
+          "type": "string",
+          "enum": [
+            "Eventual",
+            "Session",
+            "BoundedStaleness",
+            "Strong"
+          ],
+          "x-ms-enum": {
+            "name": "DefaultConsistencyLevel",
+            "modelAsString": false
+          }
+        },
+        "maxStalenessPrefix": {
+          "description": "When used with the Bounded Staleness consistency level, this value represents the number of stale requests tolerated. Accepted range for this value is 1 – 2,147,483,647. Required when defaultConsistencyPolicy is set to 'BoundedStaleness'.",
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 2147483647,
+          "format": "int64"
+        },
+        "maxIntervalInSeconds": {
+          "description": "When used with the Bounded Staleness consistency level, this value represents the time amount of staleness (in seconds) tolerated. Accepted range for this value is 1 - 100. Required when defaultConsistencyPolicy is set to 'BoundedStaleness'.",
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 100,
+          "format": "int32"
+        }
+      },
+      "required": [
+        "defaultConsistencyLevel"
+      ]
+    },
+    "DatabaseAccountProperties": {
+      "description": "Properties for the database account.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "The status of the DocumentDB account at the time the operation was called. The status can be one of following. 'Creating' – the DocumentDB account is being created. When an account is in Creating state, only properties that are specified as input for the Create DocumentDB account operation are returned. 'Succeeded' – the DocumentDB account is active for use. 'Updating' – the DocumentDB account is being updated. 'Deleting' – the DocumentDB account is being deleted. 'Failed' – the DocumentDB account failed creation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "documentEndpoint": {
+          "description": "The connection endpoint for the DocumentDB database account.",
+          "type": "string",
+          "readOnly": true
+        },
+        "databaseAccountOfferType": {
+          "description": "The offer type for the DocumentDB database account. Default value: Standard.",
+          "readOnly": true,
+          "$ref": "#/definitions/DatabaseAccountOfferType"
+        },
+        "consistencyPolicy": {
+          "description": "The consistency policy for the DocumentDB database account.",
+          "$ref": "#/definitions/ConsistencyPolicy"
+        },
+        "writeLocations": {
```

Comment:
Out of curiosity, why the discrepancy between the locations, writeLocations and readLocations between request body parameters and response body? The way things are modeled now (which may very well be correct), you will have a somewhat odd client SDK for strongly typed languages...